x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_customization.md

```diff
@@ -428,4 +523,142 @@ And additionally this request contains one or more custom rules
 Then the response should only list the custom rules as updated
 And all prebuilt rules should be listed as not updated
 And for each prebuilt rule the response should contain a message that the action is not allowed under current license
+```
+
+### Reverting a rule to Elastic version
+
+#### **Scenario: User can revert customized prebuilt rule to current Elastic version**
+
+**Automation**: 1 cypress test and 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user reverts the rule
+Then the rule should be modified to match the original Elastic rule object corresponding to its version number
+And the rule's `is_customized` value should be false
+```
+
+#### **Scenario: User can view diff between current and original Elastic rule versions in flyout**
+
+**Automation**: 1 cypress test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user navigates to that rule's details page
+And clicks the overflow actions button
+And clicks the revert rule button
```

Comment:
Let's collapse it into one line `When user click the "Revert" rule's action button on the rule's details page`.

In the general case the button might or might not be a part of overflow. Having too many UI details in the test plan would require adjusting it every time we move the button around.